src/pages/components/WebView.qml

```diff
@@ -332,6 +335,11 @@ WebContainer {
                     } else {
                         webView.findInPageHasResult = false
                     }
+                    break
+                }
+                case "embed:OpenLink": {
```

Comment:
Just add comment here that this listener is only registered for the captive portal handler.